README.md

```diff
@@ -39,9 +39,9 @@ If you want to disable data collection, please set the `disableTelemetry` argume
 Prior to running this action, a set of Azure resources and GitHub Actions are either required or optional depending on
 the arguments provided to this action.
 
-### Azure Container Registry
+### Container Registry
 
-An [Azure Container Registry](https://azure.microsoft.com/en-us/products/container-registry/) must exist that the user
+Any container registry like the [Azure Container Registry](https://azure.microsoft.com/en-us/products/container-registry/), [GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) or [Docker Container Registry](https://docs.docker.com/registry/) must exist that the user

```

Comment:
nit: let's make sure that we respect the 120 character line limit that we have going in this file (hyperlinks usually go on a line by themselves since they use up so many characters)